sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1715,6 +1718,7 @@ static void _event_cb(netdev_t *dev, netdev_event_t event)
                 break;
 #if IS_USED(MODULE_NETSTATS_L2) || IS_USED(MODULE_GNRC_NETIF_PKTQ)
             case NETDEV_EVENT_TX_COMPLETE:
+            case NETDEV_EVENT_TX_COMPLETE_DATA_PENDING:
```

Comment:
What does this event mean?

Reply:
This event is generated when the received ACK frame has the Frame Pending bit set. It needs to be indicated by radios (or SubMAC) since the ACK frame is not forwarded to the upper layer